audit_math/bravo.py

```diff
@@ -65,7 +68,9 @@ def get_expected_sample_sizes(risk_limit, contest, sample_results):
         return math.ceil((weighted_alpha + (z_w / 2.0)) / (p_w * z_w + p_l * z_l))
 
 
-def get_test_statistics(margins, sample_results):
+def get_test_statistics(
+    margins: Dict[str, Dict], sample_results: Dict[str, int]
```

Comment:
dunno if it's worth adding key/value types to the nested `Dict` in the type for `margins`